benchmarker/benchmarker.py

```diff
@@ -33,7 +33,8 @@ def parse_basic_args(argv):
     parser.add_argument("--problem_size", default=None)
     parser.add_argument("--batch_size", default=None)
     parser.add_argument("--power_sampling_ms", type=int, default=100)
-    # parser.add_argument('--misc')
+    parser.add_argument("--flops", action="store_true")
+    #parser.add_argument('--misc')
```

Comment:
```suggestion
    # parser.add_argument('--misc')
```
The right way to comment is to put a space after the comment! :P :P :P 

More serious questions: why are there two `--flop` args parts? Do both the "outer" and the "inner" benchmarker need a `--flop` flag? What does the `--flop` flag do in the inner/core benchmarker?

Other question: Do the tests pass? And did you test it with both with and without the `--flop` flag? Do they work in both cases!? :)

Reply:
Initially, I did not add it in the "inner" benchmarker but it raised an error:
benchmarker.py: error: unrecognized arguments: --flops

I tested it both with and without the --flop flag, works fine. The tests also pass OK.

Reply:
Ok. I kinda get it: the inner `--flops` doesn't do anything... it's just there so you don't get an error, right? But then how do the other non-inner benchmarking parameters not causing the error? Are the same way?

Reply:
Ok. Checked it out. It seems like they are handled in a similar way (just added and not used)... But you see what's the problem with that, right? We got to maintain 2 lists of "outer parameters" which sounds like something that could be solved somehow in a more elegant way. So you should add a `TODO(Shwerta):` there... maybe add an issue :)